src/common/libschedutil/init.c

```diff
@@ -49,13 +50,50 @@ schedutil_t *schedutil_create (flux_t *h,
     return NULL;
 }
 
+static void respond_to_outstanding_msgs (schedutil_t *util)
+{
+    int rc = 0;
+    flux_future_t *fut;
+    flux_msg_t *msg;
+    for (fut = zlistx_first (util->outstanding_futures);
+         fut;
```

Comment:
Hrm?

Reply:
Oh apologies, I misread the for-loop split over three lines.  Never mind!

Reply:
Would you prefer that I refactor this into a `while` loop or is it OK as-is?